lib/livebook_web/live/output.ex

```diff
@@ -47,6 +54,10 @@ defmodule LivebookWeb.Output do
     """
   end
 
+  defp render_output({:doctest_result, result}, %{id: id, cell_id: cell_id}) do
+    live_component(Output.DoctestComponent, id: id, result: result, cell_id: cell_id)
+  end
```

Comment:
Now that we submit multiple updates per doctest, I think we shouldn't render components, since those are just events. You can match on the output as we do here:

https://github.com/livebook-dev/livebook/blob/d92e7f98dfe727292187424bccf949da103ca54d/lib/livebook_web/live/session_live.ex#L2232-L2241

and have the `push_event`s there :)

Reply:
💯